toolchain/lower/file_context.cpp

```diff
@@ -85,14 +86,22 @@ auto FileContext::Run() -> std::unique_ptr<llvm::Module> {
   for (auto [id, _] : sem_ir_->functions().enumerate()) {
     BuildFunctionDefinition(id);
   }
+
+  // Lower function definitions for generics.
+  // TODO: this needs to be an iterator as new definitions can be added
```

Comment:
Maybe this meant to say it can't be an iterator?
```suggestion
  // This can't be an iterator, and needs to use an index, as new definitions can be added
```